src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -7,18 +8,96 @@ export default {
   component: Checkbox,
 } as Meta<CheckboxProps>;
 
-export const Default = () => (
-  <Checkbox label="example label" value="1" defaultChecked />
-);
+export const Default = () => {
+  return <Checkbox label="example label" name="example" value="1" />;
+};
+
+export const DefaultChecked = () => {
+  return (
+    <Checkbox label="example label" name="example" value="1" defaultChecked />
+  );
+};
```

Comment:
I really like the stories in this file, they're explicit in their functionality and offer an appropriately wide range of examples and use-cases. The only nit I have (and this is just a nit), is that I think these two examples should be combined. 
```suggestion
export const Default = () => (
  <>
    <Checkbox label="default checkbox" name="example" value="1" />;
    <Checkbox label="default checked checkbox" name="example" value="2" defaultChecked />
  </>
};
```